openfl-tutorials/interactive_api/MXNet_landmarks/workspace/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mxnet-cu112==1.9.0
+kaggle
```

Comment:
@ViktoriiaRomanova , i think we should add here `pandas`, `numpy` and `matplotlib` packages, may be smth else, since we use it insede notebook. So i would ask you to create three different clean virtual python virtual enviroment and test this tutorial, to resolve and understand all dependencies, which ones we should add and which ones should we remove?
This applies to envoy and director.